force-app/main/default/classes/StringGenerator.cls

```diff
@@ -0,0 +1,30 @@
+public with sharing class StringGenerator {
+
+    private List<String> my_list = new List<String>();
+
+    public StringGenerator( List<String> my_list_by_user )
+    {
+        my_list = my_list_by_user;
```

Comment:
what does myList and myListByUser represent? Just a list of values? I think name could've been better for these lists

Reply:
Noted